src/main/java/Storage.java

```diff
@@ -0,0 +1,104 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Store the tasks list into a .txt file after User exits the Duke application.
+ * It also loads up the tasks list from the selected .txt file when User starts up
+ * the Duke application.
+ */
+public class Storage {
+    private String filepath;
+    private ArrayList<Task> taskList;
+    /**
```

Comment:
There should be a extra empty line between the attribute declarations and the first function declaration.